generated/niscope/session.py

```diff
@@ -2694,16 +2718,39 @@ def fetch_into(self, wfm, relative_to=enums.FetchRelativeTo.PRETRIGGER, offset=0
         num_samples = int(len(wfm) / self._actual_num_wfms())
 
         if wfm.dtype == numpy.float64:
-            return self._fetch_into_numpy(num_samples=num_samples, wfm=wfm, timeout=timeout)
+            wfm_infos = self._fetch_into_numpy(num_samples=num_samples, wfm=wfm, timeout=timeout)
```

Comment:
I don't think the parameter name should be abbreviated either.

Reply:
Ditto about should this be in this PR